tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -90,7 +123,7 @@ Create minimal VCH
     Property Should Contain         .runtime.power_state                 poweredOn
     Property Should Contain         .runtime.upgrade_status              Up to date
 
-    Wait For VCH Initialization  name=%{VCH-NAME}-api-test-minimal
+    Get Docker Host Params    %{VCH-NAME}-api-test-minimal
```

Comment:
Still looks like it should be waiting for VCH initialization but looks like that's been replaced with `Wait Until Keyword Succeeds` for 300s.

Reply:
Related to https://github.com/vmware/vic/pull/8079/files/54c60585b6dc1bd225502b7a82b7cd453279bbaa#r197295633